server/server.go

```diff
@@ -951,7 +950,15 @@ func (s *Server) v0QueryPath(w http.ResponseWriter, r *http.Request, urlPath str
 			}
```

Comment:
☝️ I've put some of these into the `makeRego` call instead 👇 

Centralizing this, we can change fewer places to enable tracing.